Find Food Trucks SF/Find Food Trucks SF/FoodTruckDetailViewController.swift

```diff
@@ -7,25 +7,84 @@
 //
 
 import UIKit
+import MapKit
 
-class FoodTruckDetailViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
-    
+class FoodTruckDetailViewController: UIViewController, UITableViewDataSource, UITableViewDelegate, CLLocationManagerDelegate {
     
+    var foodTruck: FoodTruck!
+    var unwindSegueId: String!
+    var locationManager = CLLocationManager()
     @IBOutlet weak var foodItemsTableView: UITableView!
     @IBOutlet weak var nameLabel: UILabel!
     @IBOutlet weak var daysHoursLabel: UILabel!
     @IBOutlet weak var addressLabel: UILabel!
     @IBOutlet weak var locationDescriptionLabel: UILabel!
+    @IBOutlet weak var bookmarkButton: UIButton!
+    @IBOutlet weak var mapView: MKMapView!
     
-    var foodTruck: FoodTruck!
+    @IBAction func bookmarkButtonClicked(_ sender: UIButton) {
+        if isBookmarked() {
+            sender.setImage( UIImage(named:"unbookmarked"), for: .normal)
+            removeBookmark()
+        } else {
+            sender.setImage(UIImage(named:"bookmark"), for: .normal)
+            bookmark()
+        }
+    }
     
+    override func viewWillAppear(_ animated: Bool) {
```

Comment:
I added this to point directly to the chosen food truck location since before it was only pointing to the user location

Reply:
Good job, there definitely needs to a point to show where the user is relative to the truck.